tests/performance/performance.cpp

```diff
@@ -12,6 +28,21 @@ _perf_epoch_enter_exit()
     ebpf_epoch_exit();
 }
 
+static void
+_perf_epoch_enter_exit_alloc_free()
```

Comment:
what about other combinations? enter_alloc_exit; enter_free_exit.

Reply:
This is intended as the simplest possible operation involving epoch and memory handling. Not sure how enter_alloc_exit nor enter_free_exit would work.